presto-product-tests/conf/docker/files/presto-cli.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -euxo pipefail
```

Comment:
Nice. Didn't know something like `-o pipeline` nor `-u` exist. Seem to be very helpful.

Reply:
:) 
@arhimondr , don't be fooled, however, that this makes Bash robust. It indeed does, unless you also happen to use `if`, `||` and the like.